style.css

```diff
@@ -1,132 +1,222 @@
 @import url('https://fonts.googleapis.com/css2?family=Noto+Serif+Display:ital,wght@1,500&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Heebo:wght@300&display=swap');
 
-:root{
+:root {
     --back-color: #F8F9FA;
-    --text-col:#212529 ;
-    --light-grey:#ADB5BD;
+    --text-col: #212529;
+    --light-grey: #ADB5BD;
     --little-dark-grey: #6C757D;
     --little-light-grey: #CED4DA;
-    --heading-font:'Noto Serif Display', serif;
-    --detail-font :'Heebo', sans-serif ;
+    --heading-font: 'Noto Serif Display', serif;
+    --detail-font: 'Heebo', sans-serif;
     --bottom-shadow: 0px 15px 10px -15px var(--light-grey);
     --right-bottom-shadow: 15px -10px 15px -15px var(--little-dark-grey);
+    --all-side-shadow: 0px 0px 2px 2px var(--little-light-grey);
+    --all-side-bold-shadow: 0px 0px 10px 10px var(--little-dark-grey);
 }
-body{
+
+body {
     font-family: var(--detail-font);
     background-color: var(--back-color);
-    color:var(--text-col) ;
+    color: var(--text-col);
 }
+
 /* --navigation-- */
-.navbar{
-    padding:1rem;
-    display:flex;
+.navbar {
+    padding: 1rem;
+    display: flex;
     justify-content: space-between;
     align-items: center;
     box-shadow: var(--bottom-shadow);
 }
 
-.nav-brand{
-    font-size:xx-large;
+.nav-brand {
+    font-size: xx-large;
     font-weight: bold;
     font-family: var(--heading-font);
 }
-.nav-collapse{
-    padding:0 1rem;
+.nav-brand-link{
+    text-decoration: none;
+    color:var(--text-col);
```

Comment:
Seems you have created the navbar locally. If you have made a nav component in your CL then it's preferred to use that.

Reply:
No, I made a simple one and here I need to create another as shown in Figma sites